spec/lib/guardian_extensions_spec.rb

```diff
@@ -243,5 +243,28 @@
         end
       end
     end
+
+    describe "#can_delete_category?" do
+      alias_matcher :be_able_to_delete_category, :be_can_delete_category
```

Comment:
Trying something here to make specs clearer to read.

Reply:
not sure I follow, can you expand please?

Reply:
Yes sorry. Defining this alias allows to write `expect(staff_guardian).to be_able_to_delete_category(category)` on line 259 which reads better than `expect(staff_guardian).to be_can_delete_category(category)` or `expect(staff_guardian.can_delete_category?(category)).to eq(true)` IMO.